### ISSUE TITLE
[Issue]: Error sending notification in ios app

### ISSUE THREAD

**Issue by efrainsalasmty** [SgtBatten/HA_blueprints#192]
### Description

This is the error:

Choose at step 1: Frigate Event: Notify on new object: choice 1: Choose at step 1: choice 1: Error executing script. Error for device at pos 1: Unable to find notify service for webhook ID

Notes:
In Home assistant notifications settings (ios settings) has 2 cuestionable things the first one is Push ID "Not registered for remote notifcations"

### Version

SgtBatten/Stable.yaml

### Automation Config

```yaml
id: '1711640186992'
alias: NotifyCamaraFrontal
description: ''
use_blueprint:
  path: SgtBatten/Stable.yaml
  input:
    camera: camera.frontal
    notify_device: a15eed14ed717b319bf7de09e59fdea8
    attachment: snapshot
    debug: true
```

### Frigate Config

```yaml
mqtt:
  host: 192.168.100.11
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  stats_interval: 300

record:
  expire_interval: 10

timestamp_style:
  format: "%d/%m/%Y %H:%M:%S"
  color:
    red: 255
    green: 255
    blue: 255
  thickness: 2
  effect: shadow

cameras:
  frontal:
    ffmpeg:
      hwaccel_args: preset-vaapi
      inputs:
        - path: rtsp://admin:Prime109@192.168.100.7:554/h264Preview_01_main
          input_args: preset-rtsp-restream
          roles:
            - detect
            - record
    detect:
      width: 3840
      height: 2160
      fps: 5
      stationary:
        max_frames:
          objects:
            person: 100
    objects:
      track:
        - person
        - dog
        - bicycle
        - cat
      filters:
        cat:
          min_score: 0.8
          threshold: 0.5
        dog:
          min_score: 0.8
          threshold: 0.5
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: false
      crop: false  # Corrección del valor
      retain:
        default: 5
    record:
      enabled: true  # Corrección del valor
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 4
          mode: active_objects
        pre_capture: 5
        post_capture: 15
    ui:
      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
      # By default the cameras are sorted alphabetically.
      order: 1
      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
      dashboard: True
  porche:
    ffmpeg:
      hwaccel_args: preset-vaapi
      inputs:
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=1&subtype=0
          roles:
            - record
    detect:
      width: 1920
      height: 1080
      fps: 5
      stationary:
        max_frames:
          objects:
            person: 100
    objects:
      track:
        - person
        - dog
        - bicycle
        - cat
      filters:
        cat:
          min_score: 0.8
          threshold: 0.5
        dog:
          min_score: 0.8
          threshold: 0.5
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: false
      crop: false  # Corrección del valor
      height: 500
      retain:
        default: 5
    record:
      enabled: true  # Corrección del valor
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 4
          mode: active_objects
        pre_capture: 5
        post_capture: 15
    ui:
      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
      # By default the cameras are sorted alphabetically.
      order: 1
      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
      dashboard: True
  patio:
    ffmpeg:
      hwaccel_args: preset-vaapi
      inputs:
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=4&subtype=1
          roles:
            - detect
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=4&subtype=0
          roles:
            - record
    detect:
      width: 1920
      height: 1080
      fps: 5
      stationary:
        max_frames:
          objects:
            person: 100
    objects:
      track:
        - person
        - bicycle
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: false
      crop: false  # Corrección del valor
      height: 500
      retain:
        default: 5
    record:
      enabled: true  # Corrección del valor
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 4
          mode: active_objects
        pre_capture: 5
        post_capture: 15
    ui:
      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
      # By default the cameras are sorted alphabetically.
      order: 2
      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
      dashboard: True
  pasillo:
    ffmpeg:
      hwaccel_args: preset-vaapi
      inputs:
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=3&subtype=1
          roles:
            - detect
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=3&subtype=0
          roles:
            - record
    detect:
      width: 1920
      height: 1080
      fps: 5
      stationary:
        max_frames:
          objects:
            person: 100
    objects:
      track:
        - person
        - bicycle
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: false
      crop: false  # Corrección del valor
      height: 500
      retain:
        default: 5
    record:
      enabled: true  # Corrección del valor
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 4
          mode: active_objects
        pre_capture: 5
        post_capture: 15
    ui:
      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
      # By default the cameras are sorted alphabetically.
      order: 3
      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
      dashboard: True
  techo:
    ffmpeg:
      hwaccel_args: preset-vaapi
      inputs:
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=2&subtype=1
          roles:
            - detect
        - path: rtsp://wolty:Prime109@@192.168.100.20:554/cam/realmonitor?channel=2&subtype=0
          roles:
            - record
    detect:
      width: 1920
      height: 1080
      fps: 5
      stationary:
        max_frames:
          objects:
            person: 100
    objects:
      track:
        - person
        - dog
        - bicycle
        - cat
      filters:
        cat:
          min_score: 0.8
          threshold: 0.5
        dog:
          min_score: 0.8
          threshold: 0.5
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: false
      crop: false  # Corrección del valor
      height: 500
      retain:
        default: 5
    record:
      enabled: true  # Corrección del valor
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 4
          mode: active_objects
        pre_capture: 5
        post_capture: 15
    ui:
      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
      # By default the cameras are sorted alphabetically.
      order: 4
      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
      dashboard: True
detectors:
  cpu1:
    type: cpu
    num_threads: 2

rtmp:
  enabled: false

birdseye:
  enabled: false  # Corrección del valor
  width: 1280
  height: 720
  quality: 1
  mode: objects
```

**Comment by SgtBatten** [comment]
Interesting one. Seems you might want to reinstall the ha app on your phone.  I assume you can't send any notification to it manually using the notify service in developer tools

**Comment by Rpryning** [comment]
I am having same issue.
And Frigate Notifications blueprint is the only thing not sending any notifications at the moment.
It has been like that for some weeks.
My Frigate is running in docker and has version 0.13.2-6476F8A
The only update i can recall related is the frigate integration in HA
Please investigate as this is a showstopper

**Comment by Rpryning** [comment]
Update:
I removed the device for my IPhone in devices/mobile app
Restarted my Iphone HA App
My IPhone was added again in devices/mobile app.
Added my iphone for notification again in the blueprint and then it worked again.
Thanks.

**Comment by Drtrider** [comment]
I had a similar issue after getting a new iPhone when attempting to get it added into HA. 

Digging in the logs, and I found this entry;
`2024-06-21 15:55:26.115 ERROR (MainThread) [homeassistant.components.device_tracker] Platform mobile_app does not generate unique IDs. ID <OBJECT ID> already exists - ignoring device_tracker.user_iphone`


> Update: I removed the device for my IPhone in devices/mobile app

After going to the same settings mentioned by @Rpryning , I noticed there are two iPhones in there... Very strange. After removing both, and allowing it to re-add my device. Things are working again with the  latest version (Frigate Notifications (0.12.0.4a)) of the blueprint. Great thread there!
